packages/vite/src/node/__tests__/package.json

```diff
@@ -3,6 +3,7 @@
   "private": true,
   "version": "0.0.0",
   "dependencies": {
+    "parent": "link:./packages/parent",
```

Comment:
Could we rename the `parent` here as `@vitejs/parent`? We usually scope them to prevent accidentally matching actual npm package.

Reply:
done